core/RhythmMusicGeneration/src/main/java/org/jjazz/rhythmmusicgeneration/api/SongSequenceBuilder.java

```diff
@@ -134,6 +134,8 @@ static public class SongSequence
      * @param playbackKeyTranspose Optional chord symbols transposition (use 0 for no transposition)
      */
     public SongSequenceBuilder(SongContext sgContext, int playbackKeyTranspose)
+    // TODO #534 Probably get rid of the transpose param here, not sure if it will be used at all
+    // >>>>> public SongSequenceBuilder(SongContext sgContext) <<<<<
     {
```

Comment:
yes you can remove completly. You need to check if the song copy I had to use to create the transposed song is still needed -probably not.